dashboard/src/components/users/UserPage.vue

```diff
@@ -64,11 +72,14 @@ limitations under the License.
         'numSkills',
         'userTotalPoints',
       ]),
+      showUserTags() {
+        return !!this.$store.getters.config.userPageTagsToDisplay;
+      },
       headerOptions() {
         return {
           icon: 'fas fa-user skills-color-users',
           title: `USER: ${this.userTitle}`,
-          subTitle: `ID: ${this.userIdForDisplay}`,
+          subTitle: `ID: ${this.userIdForDisplay}${this.tags}`,
```

Comment:
consider using ``page-header``'s sub title slot-based customization in order to render labels and values slightly differently (please see ``BadgePage.vue`` for an example of ``subSubTitle`` slot usage)